cart-service/src/types/index.types.ts

```diff
@@ -38,6 +38,8 @@ export interface Configuration {
   dovetechApiKey: string;
   basicAuthPwdCurrent: string;
   basicAuthPwdPrevious: string;
+  otlpExporterEndpoint: string;
+  otlpExporterEndpointApiKey: string;
```

Comment:
Could you mark these as optional please (also `basicAuthPwdPrevious` which should also be optional)?